src/main/resources/templates/fragments/navbar.html

```diff
@@ -317,6 +317,7 @@ <h6 class="menu-title" th:text="#{navbar.sections.advance}"></h6>
                 dark_mode
               </span>
               <span class="icon-text icon-hide" id="dark-mode-text" th:data-text="#{navbar.darkmode}" th:text="#{navbar.darkmode}"></span>
+              <span class="tooltip-text">Theme</span>
```

Comment:
must be multi language supported, this text already exists in the span above as 
th:text="#{navbar.darkmode}"
and
th:text="#{navbar.settings}"